data/formats-data.js

```diff
@@ -3670,7 +3677,7 @@ exports.BattleFormatsData = {
 		tier: "Uber",
 	},
 	turtwig: {
-		randomBattleMoves: ["reflect", "lightscreen", "stealthrock", "seedbomb", "substitute", "leechseed", "toxic"],
+		randomBattleMoves: ["synthesis", "stealthrock", "bulletseed", "sandtomb", "toxic"],
```

Comment:
you probably dont need toxic here either.  Maybe Leech seed if you need a residual damage move.

Reply:
toxic+sandtomb doesnt emanate heat strats?

if it doesnt i'll get seed in instead

Reply:
oh that's sand tomb, not rock tomb.   toxic + sandtomb doesnt sound like much in LC since due to the low HP, 1/16 is literally always 1HP. and I don't thik Turtwig will last that long against anything for the residual damage to build up enough

Reply:
The intention is that sand tomb will let the tox damage build up bc it takes a lot of time to do that here, is that enough w/ synthesis chance?